src/app/qgsattributetabledialog.h

```diff
@@ -203,6 +203,13 @@ class APP_EXPORT QgsAttributeTableDialog : public QDialog, private Ui::QgsAttrib
     void updateFieldFromExpression();
     void updateFieldFromExpressionSelected();
 
+    /** Add items to the rightclick menu
+        @menu the context menu
+        @atIndex the current model index*/
+    void viewWillShowContextMenu( QMenu* menu, QModelIndex atIndex );
+    /** Zooms to the active feature*/
+    void zoomToFeature();
```

Comment:
Would `zoomToCurrentFeature()` better suit this function? As your comment states, it _Zooms to the active_ (=current) _feature_.